jenkins-jobs/pipelines/release/release-charts-pipeline.groovy

```diff
@@ -0,0 +1,189 @@
+import groovy.json.*
+import java.util.stream.*
+
+import com.cloudbees.groovy.cps.NonCPS
+
+@Library("dbz-libs") _
+
+if (
```

Comment:
As a follow-up - input parameters could be tested in a common functiona and just a list of them would be provided.